src/pages/common/standard/_gyrosense_page.c

```diff
@@ -75,11 +73,11 @@ static const char *gyro_val_cb(guiObject_t *obj, int dir, void *data)
     (void)obj;
     u8 i = (long)data;
     u8 changed = 1;
-    output[i] = GUI_TextSelectHelper(output[i], 0 , 100, dir, 1, 2, &changed);
+    mp->output[i] = GUI_TextSelectHelper(mp->output[i], 0 , 100, dir, 1, 2, &changed);
     if (changed) {
-        mp->mixer_ptr[i]->scalar = output[i] * 2 - 100;
+        mp->mixer_ptr[i]->scalar = mp->output[i] * 2 - 100;
     }
-    sprintf(tempstring, "%d%%", output[i]);
+    snprintf(tempstring, sizeof(tempstring), "%d%%", mp->output[i]);
     return tempstring;
```

Comment:
I'm not sure this is needed.  we don't really need to use snprintf everywhere.  Only places where the size is not constrained as it is here.